docs/_templates/layout.html

```diff
@@ -3,14 +3,14 @@
 {% block sidebarlogo %}
   {% if pagename == 'index' %}
     <iframe
-      src="http://ghbtns.com/github-btn.html?user=dahlia&repo=wand{{- '' -}}
+      src="http://ghbtns.com/github-btn.html?user=emcconville&repo=wand{{- '' -}}
            &type=watch&count=true&size=large"
       allowtransparency="true" frameborder="0" scrolling="0"
       width="170px" height="30px">
       {{- '' -}}
     </iframe>
     <iframe style="border: 0; margin: 15px 0; padding: 0;"
-            src="https://www.gittip.com/dahlia/widget.html"
+            src="https://www.gittip.com/emcconville/widget.html"
             width="48pt" height="22pt"></iframe>
```

Comment:
I believe this `<iframe>` should be completely removed since [Gratipay] (formerly known as Gittip) was shut down.

[Gratipay]: https://gratipay.com/